resources/authgear/templates/en/web/authflowv2/__dialog.html

```diff
@@ -8,11 +24,10 @@
   data-action="click->dialog#closeOnBackgroundClick"
   {{ template "dialog-attr" $.Ctx }}
 >
-  <div class="dialog">
```

Comment:
This `div.dialog` should be remain here. Otherwise, there is no reason for us to have a dialog template. If you want to have some levels of customisation on the dialog, I suggest you can receive a class list and add them to the div.dialog